inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -23,13 +24,14 @@ def __init__(
         self,
         protein_pdb: str,
         temperature: float = 300,
+        charged: bool = None
```

Comment:
I think some explanation of this parameter is needed in the docstring of the class `init` method.